src/InfoFields.php

```diff
@@ -87,7 +86,11 @@ private function parseData($dataString)
                 continue;
             }
 
-            preg_match('/([^:]*): ?(.*)/', $trimmedLine, $match);
+            $match = array();
+            $result = preg_match('/([^:]*): ?(.*)/', $trimmedLine, $match);
```

Comment:
A matter of style, I know: But I'd say a simple `if (!preg_match(...)) {` is enough here.